code/modules/events/space_dragon.dm

```diff
@@ -3,20 +3,19 @@
 	typepath = /datum/round_event/ghost_role/space_dragon
 	max_occurrences = 1
 	weight = 8
-	earliest_start = 70 MINUTES
-	min_players = 30
-	dynamic_should_hijack = TRUE
+	weight = 5
+	earliest_start = 35 MINUTES
 
```

Comment:
```suggestion
	weight = 5
	earliest_start = 35 MINUTES

```

Please dont remove our Min_Players and earliest start.

Reply:
Uhhh, what do you mean?

Reply:
Ah, okay